CHANGELOG.next.md

```diff
@@ -16,7 +16,7 @@ Thanks, you're awesome :-) -->
 
 #### Added
 
-* Added `search.*` fields #729
+* normalized dns fields across dns (question and answers), client, destination, host, server, source #797
```

Comment:
Please try to follow the boring changelog sentence structure when possible, especially the first word. Here we could say

```suggestion
* Added missing fields to normalize DNS-related fields across `dns` (question and answers), `client`, `destination`, `host`, `server`, and `source` #797
```

Reply:
done - updating PR shortly